dissect/target/plugins/os/unix/etc.py

```diff
@@ -0,0 +1,47 @@
+from functools import lru_cache
+from typing import Optional
+
+from dissect.target import Target
+from dissect.target.exceptions import UnsupportedPluginError
+from dissect.target.plugin import Plugin, internal
+from dissect.target.plugins.os.unix.config import (
+    ConfigurationEntry,
+    ConfigurationFilesystem,
+)
+
+
+class EtcTree(Plugin):
+    __namespace__ = "etc"
+
+    def __init__(self, target: Target):
+        super().__init__(target)
+        self.config_fs = None
+        if self.target.fs.path("/etc").exists():
+            self.config_fs = ConfigurationFilesystem(target, "/etc")
+
+    def check_compatible(self) -> None:
+        if self.config_fs is None:
+            raise UnsupportedPluginError()
```

Comment:
```suggestion
            raise UnsupportedPluginError()
```

Maybe add some message to the error